yazilim-dilleri/javascript/index.js

```diff
@@ -0,0 +1,7 @@
+var chars = ["h","e","l","l","o"," ","w","o","r","l","d"];
+var str = "";
+for(var i = 0;i < chars.length;i++) {
+  str = str + chars[i];
+}
+alert(str);
```

Comment:
```suggestion
alert(chars.reduce( (acc,currentChar) => acc + currentChar ));
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce